skimage/feature/register_translation.py

```diff
@@ -198,9 +245,9 @@ def register_translation(src_image, target_image, upsample_factor=1,
         # Center of output array at dftshift + 1
         dftshift = np.fix(upsampled_region_size / 2.0)
         upsample_factor = np.array(upsample_factor, dtype=np.float64)
-        normalization = (src_freq.size * upsample_factor ** 2)
+        normalization = (src_freq.size * upsample_factor**2)
         # Matrix multiply DFT around the current shift estimate
-        sample_region_offset = dftshift - shifts*upsample_factor
+        sample_region_offset = dftshift - shifts * upsample_factor
```

Comment:
Could you please undo all these operator spacing changes? Although they are probably for the better, they make it harder to trace the authorship of particular lines, for little gain in this case.